Python/compile.c

```diff
@@ -9704,56 +9704,77 @@ instructions_to_cfg(PyObject *instructions, cfg_builder *g)
 {
     assert(PyList_Check(instructions));
 
-    Py_ssize_t instr_size = PyList_GET_SIZE(instructions);
-    for (Py_ssize_t i = 0; i < instr_size; i++) {
+    Py_ssize_t num_insts = PyList_GET_SIZE(instructions);
+    bool *is_target = PyMem_Malloc(num_insts * sizeof(bool));
+    for (Py_ssize_t i = 0; i < num_insts; i++) {
+        is_target[i] = false;
+    }
```

Comment:
These four lines, zeroing included, can be replaced with `PyMem_Calloc(num_insts, sizeof(bool))`.